opal/models.py

```diff
@@ -122,6 +122,52 @@ def _get_field_type(cls, name):
         )
 
     @classmethod
+    def get_human_readable_type(cls, field):
```

Comment:
I mean yes and no :), so for lab tests and anything complicated we probably would want to manually override this.

What would you feel about moving the whole SerialisableFields model into a discoverable? That had defaulted to a generic Schema that takes the model as an argument. It would move all this logic out of the model while still allowing it to be overridable.

I think this is quite a good idea... However I don't want to do it now so I think leaving it where it is is more consistent.

Changing it to field_name and adding documentation sounds like a good idea